validator/webui/index.html

```diff
@@ -474,7 +474,22 @@
             }
           });
 
-          var minimumValidAmp =
+          // Interprets the parameters after the '#' in the URL.
+          var paramStrings = window.location.hash.substr(1).split('&');
+          var params = {};
+          for (var ii = 0; ii < paramStrings.length; ii++) {
+            var keyValue = paramStrings[ii].split('=');
+            if (keyValue[0].length > 0) {
+              params[keyValue[0]] = keyValue[1];
+            }
+          }
+          // If #url=<url> was provided, load it, otherwise, start with the
+          // minimum valid AMP document.
+          if (params.hasOwnProperty('url')) {
+            input.value = params['url']
```

Comment:
Thank you, I changed this so it checks the value, so now it'll only use it if it's a non-empty string.